src/components/MemberProfileEdit.tsx

```diff
@@ -109,13 +109,21 @@ const MemberProfileEdit: React.FC<MemberProfileEditProps> = ({
       },
     }));
   };
+  const githubRegex = new RegExp(
```

Comment:
This works but I meant more like a generic check if something is a URL doesn't need to be a github or leetcode url, just if it's a url in general right? Then you can use the same function for both.

Reply:
I do really want to check that they are valid leetcode and github profile urls so the widgets don't break